radio/src/translations/ru.h

```diff
@@ -1230,6 +1230,7 @@
 #define TR_VOICE_TAIWANESE              "Тайваньский"
 #define TR_VOICE_JAPANESE               "Японский"
 #define TR_VOICE_HEBREW                 "Иврит"
+#define TR_VOICE_UKRAINIAN               "Ukrainian"
```

Comment:
"Украинский"

Reply:
> "Украинский"
"Украинска" is probably more correct since "Украинска мова" is female gendered when referring to _language_.. however if we're referring to the Ukrainian _alphabet_ or _font_, those are male gendered and "Украинский" is correct..

Reply:
> > "Украинский"
> 
> "Украинска" is probably more correct since "Украинска мова" is female gendered when referring to _language_.. however if we're referring to the Ukrainian _alphabet_ or _font_, those are male gendered and "Украинский" is correct..

`radio/src/translations/ru.h` is for russian language. In Russian "Ukrainian"  translated as "Украинский", because  is male gendered in Russian language. 
So "Украинский" is correct.